src/main/resources/archetype-resources/ui.apps/src/main/content/jcr_root/apps/__projectName__/clientlibs/responsivegrid/less/grid.less

```diff
@@ -13,7 +13,40 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-@import "/libs/wcm/foundation/clientlibs/grid/grid_base.less";
-
-/* maximum amount of grid cells to be provided */
-@max_col: 12;
+ @import (reference) "/apps/weretail/clientlibs/common/variables.less";
```

Comment:
why there is a reference to "/apps/weretail" needed? We cannot expect the customer to have weretail installed by default, especially as we give the recommendation to install aem with nosamplecontent runmode.

Reply:
This archetype should provide its own copy of this file.
@pawan-mittal: can you add that file and remove those dependencies?

Reply:
@lydiapuric I forgot to remove **weretail** reference :) . Yes, I know the recommendation is to use nosamplecontent runmode. I didn't notice the reference project compared with both angular/react apps in the test phase, so now I made my changes in an angular app as well, which fixed the build issue.
@gabrielwalt I think we don't need a complete copy of the file, I just added the code which is required. Less Variable **_@grid-gutter-width_** was defined in the bootstrap variables.less file with fixed value 30px, so I moved in our grid.less instead of copying complete file.

I have committed the code. 

Thanks for your feedback.

Reply:
Please let me know your feedback.